example/code/glob_avg.py

```diff
@@ -0,0 +1,55 @@
+'''
+Creates a monthly temparature time series for a global average
+temperature, using of all Berkeley region data per time index. The
+average is of the temperature absolute value, the anomaly plus the
+base value for a time/region.  Missing data is interpolated using
+Pandas interpolate(). Data is from http://berkeleyearth.org/data/, see
+"Monthly Land" under the Gridded section, and use the "Equal Area"
+data file.
+
+'''
+import pandas as pd, gc, netCDF4, psutil 
+import csv, numpy as np
+import matplotlib.pyplot as plt
+
+url='Complete_TAVG_EqualArea.nc'
+nc = netCDF4.Dataset(url)
+clim = nc['climatology'][:,:]
+anom =  nc['temperature'][:,:]
+time =  nc['time'][:]
+
+def year(float_year):
+    year = int(float_year)
+    return year
+
+def month(float_year):
+    month = int((float_year % 1) * 12)
+    return month
+
+monidxs = np.array(list(map(month, time)))
```

Comment:
What do these values look like? I'm concerned about the month calculation, which assumes that months are equally spaced.

Reply:
The code worked fine.. month(1922.33) gives 3, month(1922.12) gives 1, etc.